apps/web/pages/_app.tsx

```diff
@@ -32,7 +33,9 @@ function MyApp(props: AppProps) {
   }
   // Use the layout defined at the page level, if available
   const getLayout = Component.getLayout ?? ((page) => page);
-
+  const calComVersion = `${pkg.version}-${
+    process.env.NEXT_PUBLIC_WEBSITE_URL === "https://cal.com" ? "h" : "sh"
+  }`;
```

Comment:
we have this in the sidebar footer, no? lets turn that into a helper function

Reply:
Yeah, thought about that. But skipped the idea because it was a small code. But let me do it now.

Reply:
Done